apps/builder/app/builder/features/sidebar-left/panels/marketplace/products.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "category": "templates",
+    "label": "CIQ Library Lite",
+    "publishedUrl": "https://launchpads-style-guide-c4i72.wstd.io",
+    "authToken": "e41a0452-a468-416c-a4da-13590db1821a",
+    "projectId": "bc5f0c98-a6b0-4544-9b3f-9ae06664c036"
```

Comment:
I don't get it. Why we put here token and project id?

Reply:
We are fetching the buidl data etc of the project in the store